src/metabase/models/setting.clj

```diff
@@ -283,6 +283,9 @@
    [:database-local LocalOption]
    [:user-local     LocalOption]
 
+   ;; should this setting be read from env vars?
+   [:can-read-from-env? :boolean]
```

Comment:
Not in this PR, but would it be cool if we put defaults directly into this schema?